src/desktop/analytics/main_layout.js

```diff
@@ -10,25 +10,9 @@ import { reportLoadTimeToVolley } from "lib/volley"
 const pageType = window.sd.PAGE_TYPE || window.location.pathname.split("/")[1]
 var properties = { path: location.pathname }
 
-if (pageType == "artwork") {
```

Comment:
Same, this is still being used and exists in Reaction: https://github.com/artsy/reaction/blob/e8cb3628d24c0d1729a74e472eaa223d7d5c1b57/src/Apps/Artwork/ArtworkApp.tsx#L46-L62